spring-cloud-stream-app-starters-docs/src/main/asciidoc/pom-dependencies.adoc

```diff
@@ -0,0 +1,30 @@
+[[pom-dependencies]]
+== App Starter POM Dependencies
+
+Following diagram highlights the important App Starter POM dependencies.
+
+image:images/starters-pom-dependencies.png[PomDependencies, scaledwidth="75%"]
+
+The dependencies are grouped in three categories:
+
+* *Core Spring Dependencies* - represent the core framework libraries such as `Spring Boot`, `Spring Integration`,
+`Spring Cloud`. The "Bill Of Materials" (BOM) patterns is actively used thought the stack to decouple the
```

Comment:
Typo: s/thought/throughout

Reply:
@artembilan @tzolov I am ok to fix it during the merge. No need to submit an updated PR